frontend/src/components/ui/stepButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from "src/components/ui/button";
+import { useStepper } from "src/components/ui/Stepper";
+
+type StepButtonProps = {
+  onNextClick?: () => void;
+};
+
+export default function StepButtons({ onNextClick }: StepButtonProps) {
+  const { nextStep, prevStep, isLastStep, isDisabledStep } = useStepper();
+  return (
+    <>
+      <div className="w-full flex justify-start gap-2 mt-8 mb-2">
+        <>
+          {isLastStep ? (
+            <Button
+              disabled={isDisabledStep}
+              onClick={prevStep}
+              variant="secondary"
+            >
+              Back
```

Comment:
What does this do when you are currently opening a channel?

Reply:
i removed back button in second step to avoid setting orders while channels is being opened